integration/build_test.go

```diff
@@ -223,7 +199,11 @@ func nowInChicago() string {
 	return time.Now().In(loc).Format("2006-01-02")
 }
 
-func failNowIfError(t *testing.T, err error) {
+type Fataler interface {
```

Comment:
"Fataler" 🤔

Reply:
yeah, that's the go idiomatic way of naming single function interfaces. Not so nice, right?